test/distributed/launcher/test_run.py

```diff
@@ -642,27 +610,6 @@ def test_init_method_tcp_with_torchelastic(self):
         )
         # nothing to validate, just make sure it runs
 
-    def test_init_method_env(self):
-        port = get_free_port()
-        with patch.dict(
-            os.environ,
-            {
-                "RANK": "0",
-                "WORLD_SIZE": "1",
-                "MASTER_ADDR": "localhost",
-                "MASTER_PORT": str(port),
-            },
-        ), patch.object(
-            sys,
-            "argv",
-            [
-                path("bin/test_script_init_method.py"),
-                "--init-method=env://",
-            ],
-        ):
-            runpy.run_path(sys.argv[0], run_name="__main__")
-            # nothing to validate, just make sure it runs
-
```

Comment:
also wondering why we're removing this one

Reply:
Both of the cases are covered in torchrun and this ones are just run the script and provide no value. The only usecase is when the script is broken, it is easier to triangulate.